components/data-list/css/_mixin.scss

```diff
@@ -29,6 +40,10 @@
   font-weight: var(--utrecht-data-list-item-value-font-weight);
   line-height: var(--utrecht-data-list-item-value-line-height);
   min-block-size: calc(var(--utrecht-data-list-item-value-line-height) * 1rem);
+  padding-block-end: var(--utrecht-data-list-item-value-padding-block-end, 0);
+  padding-block-start: var(--utrecht-data-list-item-value-padding-block-start, 0);
+  padding-inline-end: var(--utrecht-data-list-item-value-padding-inline-end, 0);
+  padding-inline-start: var(--utrecht-data-list-item-value-padding-inline-start, 0);
```

Comment:
I suspect the padding should also be configurable on the actions. What do you think?

Reply:
Yes, i've added these to actions as well. This does imply we need another vertical border. Haven't added it as it doesn't affect the Utrecht implementation. We may need to refactor this implementation after the PR to clean it up.